app/assets/stylesheets/application.css.scss

```diff
@@ -51,3 +51,33 @@ path {
   stroke: #fff;
   fill-rule: evenodd;
 }
+
+.d3-tip {
+ line-height: 1;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Properties should be ordered background, border-radius, color, font-color, font-weight, line-height, padding